js/apps/admin-ui/src/events/events.css

```diff
@@ -1,5 +1,5 @@
 .keycloak_eventsection_details {
-  --pf-c-description-list--m-horizontal__term--width: 15ch;
+  --pf-c-description-list--m-horizontal__term--width: 30ch;
```

Comment:
Any way we could do this using PF utility classes instead? All this custom CSS will be a hassle to port once we move to PF5.

Reply:
we trying to keep it to an absolute minimum, but sometimes we must the change to patternfly 5 is minimal though: https://www.patternfly.org/components/description-list#css-variables